translations/messages.fr.yaml

```diff
@@ -1 +1,24 @@
+Contribute: "Contribuez au projet\xA0:"
+CreateMyAccount: Créer mon compte
+DoB: Date de naissance
+EditMyInfo: Modifier mes informations
+From: du
+Home: Accueil
+Login: Connexion
+LoginSubmit: Je me connecte
+Logout: Déconnexion
+ProjectName: Réserve opérationnelle - Croix-Rouge Française à Paris
+RememberMe: Se souvenir de moi
+Section:
+  Volunteer: Espace bénévole
+  Organization: Espace structure
 Submit: Enregistrer
+To: au
+UserAvailabilityCurrentWeek: Mes disponibilités pour la semaine courante
+UserAvailabilityNextWeek: Mes disponibilités pour la semaine prochaine
+UserLogin: Numéro NIVOL ou Adresse e-mail
+UserIdentificationNumber: NIVOL
+Week:
+  Current: Semaine actuelle
+  Next: Semaine prochaine
+Welcome: Bienvenue
```

Comment:
can you add a last \n on all your files ? you can configure it in your IDE

Reply:
done!